Makefile

```diff
@@ -178,6 +188,23 @@ release: tar.gz
 	@echo "Add all *.aci, flanneld-* and *.tar.gz files from dist/ to the Github release"
 	@echo "Use make docker-push-all to push the images to a registry"
 
+release-tests: bash_unit
+	# Run the functional tests with different etcd versions. Currently these are the latest point releases.
+	# This list should be updated during the release process.
+	ETCD_IMG="quay.io/coreos/etcd:v3.2.7"  ./bash_unit dist/functional-test.sh
```

Comment:
what if we do a `latest` tag here instead?

Reply:
good idea